### ISSUE TITLE
License File

### ISSUE THREAD

**Issue by gkemp94** [codeforireland2/transparentwater_mobileapp#47]
After discussions of licensing and ownership, I believe this should become part of our repo at a certain point with a license file and we should fix the license in our package. Currently we explicitly state that we're licensed under a ISC license in our package.json.

Perhaps this would be a good issue for either @SmilingKat or @JohnRHarrington .

**Comment by lukekearney** [comment]
I agree. The ISC license was just a placeholder during `npm init`. It should be replaced as soon as possible. Ditto for the backend repo
